apps/antalmanac/src/components/Calendar/Toolbar/CustomEventDialog/DaySelector.tsx

```diff
@@ -30,24 +26,29 @@ const DaySelector = ({ days = [false, false, false, false, false, false, false],
             sx={{
                 display: 'flex',
                 flexDirection: 'row',
-                justifyContent: 'center',
-                padding: 5,
+                justifyContent: 'space-between',
+                marginTop: 10,
+                marginBottom: 10,
+                marginLeft: -5,
+                marginRight: -5,
             }}
         >
             {normal_days.map((day, index) => (
                 <Button
                     key={index}
                     variant={selectedDays[index] ? 'contained' : 'outlined'}
                     size="small"
+                    fullWidth
                     onClick={() => {
                         handleChange(index);
                     }}
```

Comment:
```suggestion
                    onClick={() => handleChange(index)}
```
nit: we can inline this function